torchvision/models/regnet.py

```diff
@@ -1294,7 +1294,7 @@ def regnet_y_32gf(*, weights: Optional[RegNet_Y_32GF_Weights] = None, progress:
 
 
 @register_model()
-@handle_legacy_interface(weights=("pretrained", None))
+@handle_legacy_interface(weights=("pretrained", RegNet_Y_128GF_Weights.DEFAULT))
```

Comment:
There are no `IMAGENET1K_V1` weights for this one. I'm not sure about this change; it was done purely to make the aforementioned test pass.

Reply:
There are no `IMAGENET1K_V1` weights for this one :(

I have the same concerns as you here. @jdsgomes expressed the same sentiment while speaking offline with him. I think I'll revert this to `None`.